config/deploy/staging.rb

```diff
@@ -0,0 +1,90 @@
+# server-based syntax
+# ======================
+# Defines a single server with a list of roles and multiple properties.
+# You can define all roles on a single server, or split them:
+
+server 'lib-solr-staging1', user: 'deploy', roles: %{main}
+
+# server 'example.com', user: 'deploy', roles: %w{app web}, other_property: :other_value
+# server 'db.example.com', user: 'deploy', roles: %w{db}
+
+set :branch, ENV['BRANCH'] || 'master'
+
+def config_map
+  {
+    "catalog-production" => "catalog-production",
```

Comment:
dunno what I'm talking about here but why is this going to staging?

Reply:
This config set gets placed on staging because there's a backup production index kept on the staging cloud.